package alviano.kotlin.restful.error

class NotFoundException: Exception() {



}